ray_lightning/ray_ddp.py

```diff
@@ -136,6 +137,17 @@ def __init__(self,
         else:
             self.num_gpus_per_worker = int(use_gpu)
 
+        if self.num_gpus_per_worker < 1 and num_workers > 1:
```

Comment:
Check that it's greater than 0 so it doesn't raise this message for CPU training? 😄

Reply:
Good point, updated!